spec/requests/preferences/preference_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Preferences' do
```

Comment:
Just leaving [best practices for specs](https://www.betterspecs.org/), I love this article.
You can take a quick look to have in mind for future specs.